core/src/components/input/test/highlight/input.e2e.ts

```diff
@@ -0,0 +1,283 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('input: highlights', () => {
+  test.beforeEach(({ skip }) => {
+    skip.rtl();
+  });
+
+  test.describe('input: md', () => {
```

Comment:
Why split up the tests based on mode like this? It creates a lot of duplicate markup; I feel like it would be cleaner to just have one test for each case and skip modes individually where needed.

Reply:
I think I was originally thinking that the iOS vs MD behavior was going to be more different than it actually ended up being. There are definitely tests that only apply to MD (such as the focus states), but I can combine the other tests.

Reply:
I ended up enabling focus tests for iOS too just to check that nothing unintentional is getting rendered: https://github.com/ionic-team/ionic-framework/pull/26231/commits/b8181afd2d38453f8e2d5061549fcb19fc58538e

Will update screenshots now